server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -51,9 +53,20 @@ export class ScriptoriumLambda implements IPartitionLambda {
 		this.telemetryEnabled = this.providerConfig?.enableTelemetry;
 		this.maxDbBatchSize = this.providerConfig?.maxDbBatchSize ?? 1000;
 		this.restartOnCheckpointFailure = this.providerConfig?.restartOnCheckpointFailure;
+		this.logSavedOpsTimeIntervalMs = this.providerConfig?.logSavedOpsTimeIntervalMs ?? 60000;
 	}
 
 	public handler(message: IQueuedMessage) {
+		setInterval(() => {
+			if (this.savedOpsCount > 0) {
+				Lumberjack.info("Scriptorium: Ops saved to db.", {
```

Comment:
Let's test on this for multiple docs? Is this log per document and per pod? DB metric got some issue before that the log were per request, introduced way to many logs lol